resources/webcourses/src/app/views/default/header-bar/header-bar.component.scss

```diff
@@ -21,6 +21,15 @@
   max-width: 130px;
   overflow: hidden;
   color: darkorange;
+  font-size: large;
+}
+
+#user-email {
+  max-width: 130px;
+  overflow: hidden;
```

Comment:
Specifying width, overflow, etc for both, email and user fields, might lead to headaches down the line. Let's wrap these two fields into one div e.g. `#user-container` and add it to the CSS with all common properties. This practice is known in coding as DRY: Don't Repeat Yourself.